eng/Publishing.props

```diff
@@ -45,12 +47,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ToolsetAssetsToPublish Include="$(ArtifactsNonShippingPackagesDir)*.zip" />
+    <ToolsetAssetsToPublish Include="$(ArtifactsNonShippingPackagesDir)dotnet-toolset*.zip" />
```

Comment:
This is so that `PublishToolsetAssets` only publishes toolset assets. Previously, they were the only contents in this directory. Now, the Installer processes also put packages here. The Installer publishing is within `PublishSdkAssetsAndChecksums`.